block/manager.go

```diff
@@ -125,7 +125,16 @@ func (m *Manager) SetDALC(dalc da.DataAvailabilityLayerClient) {
 }
 
 func (m *Manager) AggregationLoop(ctx context.Context) {
+	genesisTime := m.genesis.GenesisTime
+	height := m.store.Height()
+	if height == 0 && time.Until(genesisTime) > 0 {
```

Comment:
In both cases, we should log information that node is waiting for correct time to produce a block. Especially in case of `GenesisTime`, detailed information should be logged (how long it will wait of when it will start creating blocks).

Reply:
I added m.log.Info() to log the wait time.